Sources/TransferMethod/ListTransferMethodViewController.swift

```diff
@@ -65,7 +56,10 @@ public final class ListTransferMethodViewController: UITableViewController {
 
     // MARK: - Transfer method list table view dataSource and delegate
     override public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return presenter.numberOfCells
+        let numberOfCells = presenter.numberOfCells
+        //viewNoResults.isHidden = presenter.isLoading ? true : numberOfCells > 0
+
+        return numberOfCells
```

Comment:
You could revert this change to be this `return presenter.numberOfCells`

Reply:
Yea, same here copied from another pr